sympy/utilities/autowrap.py

```diff
@@ -138,13 +138,13 @@ def wrap_code(self, routine, helpers=[]):
         oldwork = os.getcwd()
         os.chdir(workdir)
         try:
-            sys.path.append(workdir)
+            sys.path.append(os.getcwd())
```

Comment:
ah, providing the full path to `tempdir` also works, it was just when i was providing the relative path that I ran into problems.